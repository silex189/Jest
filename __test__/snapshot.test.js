import { getCharacter } from '../snapshot';
import rick from '../rick.json';


describe('Es hora de las instantaneas', () => {

    test('snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    })

    /* test('siempre fallara la instantanea', () => {
        
        const user = {
            createAt: new Date(),
            id: Math.floor(Math.random() * 20 ),
        }

        expect(user).toMatchSnapshot();
    }) */

    test('Tenemos una excepcion dentro del codigo', () => {
        
        const user = {
            id: Math.floor(Math.random() * 20 ),
            name: "Oscar Barajas",
        }
        expect(user).toMatchSnapshot({
            id: expect.any(Number)
        })
    })
})