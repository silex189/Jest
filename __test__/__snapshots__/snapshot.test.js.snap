// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las instantaneas Tenemos una excepcion dentro del codigo 1`] = `
Object {
  "id": Any<Number>,
  "name": "Oscar Barajas",
}
`;

exports[`Es hora de las instantaneas snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;
